detox/ios/Detox/Invocation/Element.swift

```diff
@@ -124,6 +124,10 @@ class Element : NSObject {
 		view.dtx_longPress(at: point, duration: duration)
 	}
 	
+	func longPress(at normalizedPoint: CGPoint, duration: TimeInterval, dragToElement targetElement: Element, normalizedTargetPoint: CGPoint, speed: CGFloat, holdForDuration lastHoldDuration: TimeInterval) {
+		view.dtx_longPress(at:normalizedPoint, duration:duration, target:targetElement, normalizedTargetPoint:normalizedTargetPoint, speed:speed, lastHoldDuration:lastHoldDuration)
```

Comment:
> target:targetElement

I don't think this is correct. You need to send `targetElement.view`.

Reply:
Correct, you are right.